docs/technical/installing-streetmix.rst

```diff
@@ -47,6 +47,11 @@ You may already have some of these prerequisites installed. Skip or update the p
       mkdir -p /data/db
       chmod 777 /data/db
 
+.. warning::
+
+   These instructions are now out-of-date on Mac OS X 10.15 (Catalina), as those paths are no longer writeable. You will need to set the database path to another location, such as ``/System/Volumes/Data/data/db``, and run MongoDB using the ``--dbpath`` option to point to the new location.
```

Comment:
IMO would be nice to have an explicit command to run here, I assume we can update the lines prior to this with `/System/Volumes/Data/data/db` and then running mongo is as simple as ```mongod --dbpath /System/Volumes/Data/data/db```, right?

Reply:
Yeah, I agree a command will be nice, but the problem is that I have no guarantee that the explicit command will be the same for everyone. I happened to move my `/data/db` to the `/System` location under the advice of a StackOverflow post, not because MongoDB tells people to do that (that I can see). New installations seem to use a different default database directory entirely.

If the goal is to migrate off MongoDB, then any documentation we write on it will be removed soon anyway, so it's not the highest of priority for me to think through the ramifications of MongoDB documentation.